capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -0,0 +1,288 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.clients;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import java.util.*;
+
+/**
+ * This client is in charge of interacting with metricsServlet and the metrics entities.
+ */
+public class MetricsClient {
+
+    /**
+     * Empty constructor since we don't need any variables, we only use this class for the methods.
+     */
+    public MetricsClient() {}
+
+    private static DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+
+    /**
+     * Gets the current week's page views for one restaurant by specifying a restaurantKey.
+     * @param restaurantKey restaurant key of the restaurant we want to get views.
+     * @param restaurantName restaurant name in case no entity exists for this week's page views.
+     * @return returns a Map<String, Object> of the form:
+     *          {
+     *              "year": <String, the year number>,
+     *              "week": <String, this week's number in relation to the year.>,
+     *              "count": <String, amount of page views the restaurant had.>
+     *          }
+     */
+    public Map<String, Object> getCurrentPageViews(String restaurantName, String restaurantKey) {
+        Map<String, Object> resultMap = new HashMap<>();
+
+        Calendar calendar = Calendar.getInstance();
```

Comment:
I might go back to this comment after I see your unit test, but I predict that in order to test properly, we'll want to inject something here. When code relies on time, it's USUALLY best for the test to allow controlling the time (usually just setting a permanent "reference time" for the test, unless you do something more complex).